app/controllers/account_controller.rb

```diff
@@ -28,35 +28,18 @@
 #  no_email::           <tt>(L V .)</tt>
 #  api_keys::           <tt>(L V .)</tt>
 #
-#  ==== Admin utilities
-#  turn_admin_on::      <tt>(R . .)</tt>
-#  turn_admin_off::     <tt>(R . .)</tt>
-#  switch_users::       <tt>(R V .)</tt>
-#  add_user_to_group::  <tt>(R V .)</tt>
-#  create_alert::       <tt>(R V .)</tt>
-#  destroy_user::       <tt>(R . .)</tt>
-#  blocked_ips::        <tt>(R V .)</tt>
-#
 #  ==== Testing
 #  test_autologin::     <tt>(L V .)</tt>
 #
 ################################################################################
 class AccountController < ApplicationController
   before_action :login_required, except: [
-    :email_new_password,
-    :login,
-    :logout_user,
-    :reverify,
-    :send_verify,
-    :signup,
-    :verify,
+    :new,
+    :create,
     :welcome
   ]
```

Comment:
Is this still needed/desirable? it says that login is required before all actions, then excepts every remaining action in this controller.